pkg/services/datasources/models.go

```diff
@@ -191,12 +191,15 @@ type UpdateDataSourceCommand struct {
 	IsDefault       bool              `json:"isDefault"`
 	JsonData        *simplejson.Json  `json:"jsonData"`
 	SecureJsonData  map[string]string `json:"secureJsonData"`
-	Version         int               `json:"version"`
 	UID             string            `json:"uid"`
 	// swagger:ignore
-	APIVersion string `json:"apiVersion"`
+	APIVersion string `json:"apiVersion,omitempty"`
 	// swagger:ignore
-	IsPrunable bool
+	IsPrunable bool `json:"-"`
+	// Everything above is identical in AddDataSourceCommand
+
+	// The previous version -- used for optimistic locking
+	Version int `json:"version"`
```

Comment:
This has no real changes to the model, it only changes the order so that AddDataSourceCommand and UpdateDataSourceCommand share the same field order.

NOTE: the only difference between them is that version exists in UpdateDataSourceCommand